### ISSUE TITLE
add a language/framework column in README

### ISSUE THREAD

**Issue by nikhita** [OpenSourceHelpCommunity/ContributingToMyProject#11]
Thank you, @arpitgogia for the idea! Exported from #10.

/cc @tapasweni-pathak @vaibhavsingh97 @amarlearning @jarifibrahim @anubhakushwaha

**Comment by nikhita** [comment]
We can obviously have both in a single column. :+1:

**Comment by vaibhavsingh97** [comment]
Yeah that's nice idea!! :heart_eyes:  Instead of writing `python` as an language, We use logo in that cell.

**Comment by vaibhavsingh97** [comment]
@nikhita what's the progress?? Should we proceed with the language section in the readme

**Comment by nikhita** [comment]
@vaibhavsingh97 Yes, that would be good. I am going to be label this as a `good first issue` and `hacktoberfest`.

**Comment by nik72619c** [comment]
Is the issue free to work on?
